Assets/ScriptableObjects/Items/HatBarrel.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   id: Hatapult
   displayName: Hat-apult
   secretName: tipper
-  displayDescription: Shoot hats that get stuck to your enemies!
+  displayDescription: Shoot hats that stick to your enemies!
```

Comment:
Maybe we should just rewrite this description to `Shoot hats that bounce around!` as this description is before they even riccochet bounced on the ground 😅